### PR TITLE
Empty rows bug

### PR DESCRIPTION
**Pull request by RoriCremer**
Two bug fixes:

Add safety rails in case there are no studies or datasets

I got too greedy on the cleanup...this passes the dataset request information to the created dataset object/store so that the preview page can show the appropriate info

### CODE REVIEW

File: src/components/DatasetPreviewView.jsx [hunk 1]
```diff
@@ -155,12 +155,12 @@ export class DatasetPreviewView extends React.PureComponent {
                 </div>
               ) : (
                 <div className={classes.values}>
-                  {createdDataset && createdDataset.dataset.name}
+                  {createdDataset && createdDataset.datasetRequest.name}
```
Comment (ahaessly) [inline src/components/DatasetPreviewView.jsx]: do we need to check createdDataset.datasetRequest here and on line 163?

File: src/sagas/repository.js [hunk 2]
```diff
@@ -148,6 +148,7 @@ export function* getDatasets({ payload }) {
       authGet,
       `/api/repository/v1/datasets?offset=${offset}&limit=${limit}&sort=${sort}&direction=${direction}&filter=${filter}`,
     );
+    console.log(response);
```
Comment (ahaessly) [inline src/sagas/repository.js]: remove debug message

**Review by ahaessly — APPROVED** [review]
2 comments

File: src/components/DatasetPreviewView.jsx (hunk 1 above)
Comment (RoriCremer) [reply src/components/DatasetPreviewView.jsx]: hmmmm--yes. It feels more likely that createdDataset would be the thing that gets lost, but no reason not to have the extra safety rails

**Review by thathert — APPROVED** [review]
LGTM. yes I think createdDataset is needed.

**Pull request merged**